.travis.yml

```diff
@@ -2,12 +2,16 @@ language: php
 
 php:
   - 7.1
+  - 7.2
 
 cache:
   directories:
     - $HOME/.composer/cache/files
     - $HOME/.composer/cache/vcs
 
+before_install:
+  - composer self-update 1.10.21
```

Comment:
Added this in before_install stage. Please let me know if I need to address it in a better way. @hearst-feng This is ready for another look